src/config/configUser.js

```diff
@@ -0,0 +1,121 @@
+export const userFields = [
+  {
+    key: 'cuisine',
+    label: 'Favorite cuisine',
+    scope: 'public',
+    schemaType: 'enum',
+    enumOptions: [
+      { option: 'italian', label: 'Italian' },
+      { option: 'chinese', label: 'Chinese' },
+      { option: 'thai', label: 'Thai' },
+    ],
+    showConfig: {
+      displayInProfile: true, // key might change
+    },
+    saveConfig: {
+      displayInSignUp: true, // key might change
+      required: true,
+    },
+    userTypeConfig: {
+      limitToUserTypeIds: true,
+      userTypeIds: ['a', 'b', 'c'],
+    },
+  },
+  {
+    key: 'canCook',
+    label: 'Can you cook?',
+    scope: 'public',
+    schemaType: 'boolean',
```

Comment:
Am I correct that we have everything available in the codebase for supporting **boolean** and **long** schema types for user fields?

We haven't added those schematypes for the listing fields, because we have not created filter components for them yet. However, user fields don't need filter components.

Reply:
Yes, both SectionDetailsMaybe and CustomExtendedDataField support boolean and long already.

Reply:
I pinged Juho & Thoma about this possibility (but I don't expect that we would reveal long&boolean through the Console in a different pace with listing fields - it probably makes a better release story to have them both work the same way)

Reply:
Ok. Would it still make sense to have that capability on the template side, even if Console does not yet support it? It could be helpful for customizers.

Reply:
Yes, definitely!
This was more about making a note that after this PR is merged, it only needs Console changes to take boolean and long schema types into use for user fields.